approved_tags.yaml

```diff
@@ -0,0 +1,101 @@
+---
+- "accessibilità"
+- "machinelearning"
+- "android"
+- "api"
+- "apl"
+- "arte"
+- "domanda"
+- "assembly"
+- "audio"
+- "libro"
+- "browsers"
+- "c"
+- "c++"
+- "clojure"
+- "compilatori"
+- "informaticateorica"
+- "crittografia"
+- "criptovalute"
+- "css"
+- "politica"
+- "società"
+- "etica"
+- "culturait"
+- "database"
+- "debugging"
+- "design"
+- "devops"
+- "sistemidistribuiti"
+- "dotnet"
+- "istruzione"
+- "elixir"
+- "elm"
+- "emacs"
+- "erlang"
+- "evento"
+- "metodiformali"
+- "fortran"
+- "freebsd"
+- "giochi"
+- "go"
+- "grafica"
+- "hardware"
+- "haskell"
+- "storia"
+- "ios"
+- "java"
+- "javascript"
+- "lavoro"
+- "legge"
+- "linux"
+- "lisp"
+- "lua"
+- "mac"
+- "matematica"
+- "meta"
+- "ml"
+- "mobile"
+- "netbsd"
+- "reti"
+- "networking"
+- "nodejs"
+- "objectivec"
+- "openbsd"
+- "sistemioperativi"
+- "osdev"
+- "pdf"
+- "performance"
```

Comment:
Direi di tradurre in `prestazione` (o `prestazioni`).

Reply:
Non rende così bene in italiano però io lascerei performance